### ISSUE TITLE
CloudFormation Stack Not Working

### ISSUE THREAD

**Issue by chase2981** [evanchiu/stu#3]
I am not able to deploy this repository's AWS Lamda CloudFormation stack due to the following errors:

```
The runtime parameter of nodejs6.10 is no longer supported for creating or updating AWS Lambda functions. We recommend you use the new runtime (nodejs8.10) while creating or updating functions. (Service: AWSLambdaInternal; Status Code: 400; Error Code: InvalidParameterValueException; Request ID: 073db49b-79fa-11e9-b808-db8c6a6d57a3)
```

As shown here:

<img width="1680" alt="Screen Shot 2019-05-18 at 11 59 52 PM" src="https://user-images.githubusercontent.com/10983569/57978351-60be9180-79c9-11e9-8973-60ad7d40c1e9.png">

Just letting you know...

**Comment by evanchiu** [comment]
Hello @chase2981, thanks for the note!

I'm personally in the middle of a move, so I don't have a development environment ready to fix this in the next few days.

I had tried the upgrade just before the 4/29 deprecation deadline, and found that AWS CodeStar deployment permissions had changed, so I wasn't able to test the update.  It should be similar to the updates I made in [serverless-galleria](https://github.com/evanchiu/serverless-galleria/commit/58824ed882a5c312edb441bc5a2405b03efd3fa7).

Thanks!

**Comment by evanchiu** [comment]
Hey @chase2981!  I've published version 1.0.3, which uses `nodejs10.x`, so the deployment should work for you now.

See [stu](https://serverlessrepo.aws.amazon.com/#/applications/arn:aws:serverlessrepo:us-east-1:233054207705:applications~stu) on the AWS Serverless Application Repository
